content/cpp/concepts/vectors/vectors.md

```diff
@@ -68,6 +68,28 @@ Due to the dynamic nature of vectors, it can be hard to keep track of the number
 std::cout << order.size();
 ```
 
+### `.capacity()`
+
+It is used to retrieve the current capacity of a `std::vector`. The capacity of a vector represents the number of elements it can hold without needing to allocate additional memory. The vector may have more capacity than its current size to reduce the need for frequent reallocations and copying when elements are added. 
```

Comment:
```suggestion
`.capacity()` is used to retrieve the current capacity of a `std::vector`. The capacity of a vector represents the number of elements it can hold without needing to allocate additional memory. The vector may have more capacity than its current size to reduce the need for frequent reallocations and copying when elements are added. 
```